lib/tasks/build.js

```diff
@@ -172,6 +185,15 @@ module.exports.sass = function(gulp, config) {
 
 		const combinedStream = combine.obj(
 			gulp.src(src),
+			inject.prepend(`$obt_env: '${config.env}';`),
```

Comment:
obt comes with a custom gulp plugin called prefixer, you could use that one instead of bringing in a new dependency